railties/lib/rails/application/default_middleware_stack.rb

```diff
@@ -71,7 +71,6 @@ def build_stack
           end
 
           unless config.api_only
-            middleware.use ::ActionDispatch::Flash
```

Comment:
Hmm, actually had a brain fart. I need to leave it for now, otherwise it will break app if one do `config.middleware.insert_after(Flash, MyMiddleware)`. Will modify